packages/components/src/ui/context/stories/ComponentsProvider.stories.js

```diff
@@ -19,20 +19,39 @@ export default {
 	title: 'G2 Components (Experimental)/ContextSystemProvider',
 };
 
+const outerContext = {
+	Card: {
+		isRounded: false,
+		elevation: 15,
+	},
+	CardBody: {
+		as: 'a',
+		href: 'https://wordpress.org',
+		style: {
+			display: 'block',
+		},
+	},
+};
+
 const innerContext = {
 	Card: {
-		css: {
+		style: {
 			background: 'white',
```

Comment:
Should this be something other than white, something we can actually see the difference of?

Reply:
Good point, I changed it to a shade of green in https://github.com/WordPress/gutenberg/pull/32743/commits/aff830ca84ca7b21bb9d1f4e781332961c3cf07b